docs/docs/adding-a-shopping-cart-with-snipcart.md

```diff
@@ -0,0 +1,201 @@
+
+Building on the stacking ring example, suppose that you wanted to give your customer a choice between available sizes. You would do this by adding a custom field with a name and options.
+
+```jsx
+<section>
+  <h2>Silver Stacking Ring</h2>
+  <p>$19.99</p>
+  <p>Wear one or seventeen! These rings are fun to mix and match.</p>
+  <button
+    class="snipcart-add-item"
+    data-item-id="silver-stacking-ring"
+    data-item-price="19.99"
+    data-item-url="/"
+    data-item-name="Silver Stacking Ring"
+    <!--
+    highlight-start
+    --
+  >
+    data-item-custom1-name="Size"
+    data-item-custom1-options="6|6.5|7|7.5|8|8.5|9">
+    <!-- highlight-end -->
+    Add to cart
+  </button>
+</section>
+```
+
+You can add multiple custom fields by incrementing the index of the `data-item-custom` attribute. Perhaps you want customers to have the ability to mark each item in their cart as a gift.
+
+```jsx
+<section>
+  <h2>Silver Stacking Ring</h2>
+  <p>$19.99</p>
+  <p>Wear one or seventeen! These rings are fun to mix and match.</p>
+  <button
+    class="snipcart-add-item"
+    data-item-id="silver-stacking-ring"
+    data-item-price="19.99"
+    data-item-url="/"
+    data-item-name="Silver Stacking Ring"
+    data-item-custom1-name="Size"
+    data-item-custom1-options="6|6.5|7|7.5|8|8.5|9"
+    <!--
+    highlight-start
+    --
+  >
+    data-item-custom2-name="This is a gift" data-item-custom2-type="checkbox">
+    <!-- highlight-end -->
+    Add to cart
+  </button>
+</section>
+```
+
+### Selling digital products
+
+Snipcart enables the sale of digital goods. To sell a file, you'll need to upload it to your Snipcart dashboard and then add the resulting guid as the value of the `data-item-file-guid` attribute to your product's markup. You can specify a file access expiry in days and a maximum number of downloads per order from the dashboard.
+
+```jsx
+<section>
+  <h2>Silver Stacking Ring</h2>
+  <p>$19.99</p>
+  <p>Wear one or seventeen! These rings are fun to mix and match.</p>
+  <button
+    class="snipcart-add-item"
+    data-item-id="silver-stacking-ring"
+    data-item-price="19.99"
+    data-item-url="/"
+    data-item-name="Silver Stacking Ring"
+    <!--
+    highlight-start
+    --
+  >
+    data-item-file-guid="your-digital-product-guid">
+    <!-- highlight-end -->
+    Add to cart
+  </button>
+</section>
+```
+
+## Customizing the cart
+
+Using Snipcart allows you to retain nearly complete control over your customers' experience on your e-commerce site. You can configure and customize the cart behavior as well as the product options. Look through your [Snipcart account settings](https://app.snipcart.com/dashboard/account/settings) to change things like currency, shipping options, and email templates.
+
+### Preventing automatic popups
+
+By default, the shopping cart will pop up every time a customer adds a product. To prevent this bahavior, set the value of `autopop` to `false` in your `gatsby-config.js` file.
```

Comment:
```suggestion
By default, the shopping cart will pop up every time a customer adds a product. To prevent this behavior, set the value of `autopop` to `false` in your `gatsby-config.js` file.
```